src/Components/Facility/ConsultationForm.tsx

```diff
@@ -275,9 +275,10 @@ export const ConsultationForm = (props: any) => {
             special_instruction: res.data.special_instruction || "",
             weight: res.data.weight ? res.data.weight : "",
             height: res.data.height ? res.data.height : "",
-            bed: res.data?.current_bed?.bed_object?.id || null,
+            bed: res.data?.current_bed?.bed_object || null,
```

Comment:
don't forget to make the respective changes like, types, also check other places